.github/workflows/win-package-test.yml

```
@@ -378,7 +378,12 @@ jobs:
               throw "choco install failed!"
             }
             $env_vars = Get-ItemPropertyValue -Path "HKLM:\SYSTEM\CurrentControlSet\Services\splunk-otel-collector" -Name "Environment"
-            $env_vars += "MY VAR WITH SPACES=my value"
+            $test_env_vars = @(
+              "MY_VAR_WITH_SPACES=my value",
```

Should we keep/add a test for var names with spaces?
```suggestion
              "MY VAR WITH SPACES=my value",
```

---

Yes, can I blame Copilot? :)